tests/test_scope.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
```

Comment:
I don’t think we need the `use strict` if we require Node 8+?

Reply:
Huh, I always used it in CJS modules though not in ES6 modules. That could be an obsolete practice, though. Not sure!

I found this which doesn't seem conclusive, but suggests it's still doing something: http://imaginativethinking.ca/what-the-heck-is-node-modules-strict-by-default/